runtime/service_integ_test.go

```diff
@@ -583,7 +583,7 @@ func TestLongUnixSocketPath_Isolated(t *testing.T) {
 	// default location we store state results in a path like
 	// "/run/firecracker-containerd/default/<vmID>" (with len 112).
 	const maxUnixSockLen = 108
-	vmID := strings.Repeat("x", 72)
+	vmID := strings.Repeat("x", 76)
```

Comment:
why 76?

Reply:
That's the original number before my change (#414). It could be shorter though.